utils/cw721-utils.ts

```diff
@@ -167,7 +159,7 @@ function buildQueryCommands(program: Command, client: DesmosClient) {
             console.log(`Querying owner of token id ${options.tokenId}`);
             const owner = await client.queryContractSmart(options.contract, {
                 owner_of: { token_id: options.tokenId, include_expired: options.includeExpired },
-            } as QueryMsgFor_Empty);
+            } as QueryMsgForEmpty);
```

Comment:
Due to cw-nfts does not apply cw_serde and new json schema yet, it remains the same way as before.